atlasdb-jdbc/src/main/java/com/palantir/atlasdb/keyvalue/jdbc/JdbcKeyValueService.java

```diff
@@ -641,6 +643,22 @@ public Void apply(DSLContext ctx) {
     }
 
     @Override
+    public void deleteRange(TableReference tableRef, RangeRequest range) {
```

Comment:
Looks extremely similar to (exactly the same as?) the Abstract impl. Also we don't support JDBC any more.